build_debian.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+echo "================================================"
+echo " Attention - Building snarkOS from source code."
+echo " This will request root permissions with sudo."
+echo "================================================"
+
+# Install Debian dependencies
+
+sudo apt-get update
+sudo apt-get install -y \
+	build-essential \
+	curl \
+	gcc \
+	libssl-dev \
+	make \
+	pkg-config \
+	tmux \
+	xz-utils \
```

Comment:
add `sudo`, it's not installed in the default debian docker images. It's also missing from ubuntu docker images, so please add it there too.